app/models/invenio_rdm_record_converter.rb

```diff
@@ -375,6 +375,15 @@ def format_publication_date(publication_date)
     normalize_date(publication_date)
   end
 
+  def format_dates(dates)
+    formatted_dates = dates.map do |date|
+        next if date.blank?
+        {"date": normalize_date(date), "type": {"id": "created"}, "description": "When the item was originally created."}
```

Comment:
minor: What about smaller line lengths + filter-map pattern? So something like

```ruby
def format_dates(dates)
  dates.select(&:blank?).map do |date|
    {
      "date": normalize_date(date), 
      "type": {"id": "created"}, 
      "description": "When the item was originally created."
    }
  end
end
```

Reply:
I'm fine with smaller line lengths. At some point it might be nice to add a style linter. I actually kind of like `next if` on it's own line, but this is better.